client/src/components/ButtonExampleAnimated.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Button } from 'semantic-ui-react';
+
+const ButtonExampleAnimated = (props) => {
+  //on hover should toggle to .btn-flashy book-cta
```

Comment:
When I load the page, the button does a cool thing with the text when I hover! nice work!!